README.md

```diff
@@ -25,7 +23,7 @@ The following commonly requested improvements over eCamp v2 are already implemen
 
 eCamp v3 is made up of a backend based on the PHP framework Laminas (which offers a HAL JSON API) as well as a Vue.js frontend and some other smaller services.
 
-#### Want to help?
+### Want to help?
```

Comment:
If you change this, please also change the corresponding headings in CONTRIBUTING.md